wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/zh.md

```diff
@@ -117,27 +90,59 @@ lazer 内的计分系统与 ScoreV2 系统类似，并且这套系统会把所
 
 ![](img/notelock.gif)
 
-在密集的排列中，从失误中恢复将会更简单。
+在密集的排列中，能更轻松地从失误后恢复过来。
 
 |  |  |
 | :-- | :-: |
 | 旧版本不兼容 | ![是][true] |
-| Classic 模组能够还原旧版本的功能效果 | ![是][true] |
+| Classic 模组能够还原旧版本的效果 | ![是][true] |
 | 这个更改是刻意的 | ![是][true] |
-| 这个更改需要再三考虑 | ![否][false] |
+| 这个更改需要进一步的考虑 | ![否][false] |
 
 ### 滑条头的判定包含准确率
 
 ![](img/slideracc.gif)
 
-出于历史原因，在 lazer 之前，玩家点击滑条头时，只要判定在 50/MEH 的区间内，就可以获得完美的分数。但是这种设定不太符合一款音乐节奏游戏的定位。展望未来，点击滑条头的判定会更严格。
+在 lazer 之前，玩家只要在 50/MEH 的判定区间内点击滑条头，就可以获得完美的分数。这是出于一些历史原因而设定的，不太适合一款节奏类音乐游戏。因此，滑条头的判定以后会更严格。
+
+|  |  |
+| :-- | :-: |
+| 旧版本不兼容 | ![是][true] |
+| Classic 模组能够还原旧版本的效果 | ![是][true] |
+| 这个更改是刻意的 | ![是][true] |
+| 这个更改需要进一步的考虑 | ![否][false] |
+
+### 滑条头更宽松
+
+当玩家过早点击滑条头时，即使光标在滑条开始滑动之前离开了滑条球，滑条也能立刻进入跟随状态。
+
+![](img/slider-early-leniency.gif)
+
+此外，当过晚点击滑条头时，滑条球已经走过的滑条点或折返点也会判定为完成。
+
+![](/wiki/shared/news/2023-12-18-osulazer-updates-preparing-for-ranked-play/slider-late-leniency.jpg "在此区间内点击会造成miss（更新前）")
+
+Youtube 上的官方 [lazer 更新日志](https://www.youtube.com/watch?v=xTRwM3zhhj0&t=243s)有更详细的解释。
```

Comment:
Can I use mirrored versions of this video from other websites?
for some reason, China cannot access youtube.

我们有可能用别的网站的搬运吗，比如说b站的

Reply:
Certainly. Mirrors should already have been used, I just too lazy to find since it doesn't have a channel focusing on lazer's updates.
当然可以，就应该用搬运视频的，不过还没有专门搬运lazer的号，我只是懒得找而已（

Reply:
What I care more about is what the wiki maintainers think, because it is not an official source after all.
So I think it’s understandable that we are not allowed to change it.

主要是看wiki维护者怎么想，毕竟不是官方来源，所以不让改也能理解

Reply:
as discussed in https://discord.com/channels/188630481301012481/218677502141399041/1223535920469250048, feel free to add the mirror as long as the original link is also present

Reply:
加了
added the mirror links and put the youtube's in the footnote.